### ISSUE TITLE
Comics: Indicate comic book archives in thumbnail view

### ISSUE THREAD

**Issue by bbappserver** [hydrusnetwork/hydrus#1541]
The new CBZ support is cool, but it's now difficult to tell what is and is not a comic without actually selecting it, since the cover page replaces the archive placeholder image.

I don't know if doing a fancy stacked images thumbnail is already a priority and will make this redundant, but a quick fix would be to put another icon like the inbox or notes icon.

**Comment by brachna** [comment]
Agree. Sometimes I delete uninteresting images but in case of CBZ interesting ones can be inside CBZ and I have to double check that CBZ wasn't selected. An icon would make it much easier.
